addons/packages/knative-serving/1.0.0/README.md

```diff
@@ -1,91 +1,115 @@
+
+Create a data values file.
+
+```shell
+cat <<EOF >knative-values.yaml
+domain:
+  type: real
+  name: example.com
+EOF
+```
+
+Install Knative-Serving
+
+```shell
+tanzu package install knative-serving \
+–package-name knative-serving.community.tanzu.vmware.com \
+–version 1.0.0 \
+–values-file knative-values.yaml
+```
 
-* Knative Serving version: 1.0.0
+## Options
 
-## Configuration
+### Package configuration values
 
 | Config | Values | Default | Description |
 |--------|--------|--------|-------------|
-| namespace | any namespace | `knative-serving`| Namespace where you want to install knative |
-| domain.type | real, sslip.io, nip.io | nip.io | Type of DNS resolution to use for your knative services. We can use real dns, in which case, you need to provide a domain.name or else use sslip.io or nip.io |
-| domain.name | any domain name | empty | If you have a valid domain, make sure that it's properly configure to your ingress controller |
-| ingress.external.namespace | any namespace | `projectcontour` | Namespace of the ingress controller for external services |
+| namespace | any namespace | `knative-serving`| Namespace where you want to install knative. |
+| domain.type | real, sslip.io, nip.io | nip.io | Type of DNS resolution to use for your Knative services. If you use real DNS, you need to provide a domain.name or else use sslip.io or nip.io. |
+| domain.name | any domain name | empty | If you have a valid domain, make sure that it's properly configured to your ingress controller. |
+| ingress.external.namespace | any namespace | `projectcontour` | Namespace of the ingress controller for external services. |
 | ingress.internal.namespace | any namespace | `projectcontour` | Namespace of the ingress controller for internal services. If you don't want to have internal services separated from external, use the same namespace for both. |
-| tls.certmanager.clusterissuer | `ClusterIssuerName` | empty | Name of a cert-manager ClusterIssuer to provide wildcard certificates for your cluster |
+| tls.certmanager.clusterissuer | `ClusterIssuerName` | empty | Name of a cert-manager ClusterIssuer to provide wildcard certificates for your cluster. |
 
-### Installation
+### Application configuration values
 
-The Knative Serving package requires use of Contour for ingress. To successfully install and use the Knative Serving package, you must first install Contour.
+There are no additional application configuration options that can be passed to the Knative-Serving application at this time.
 
-```shell
-tanzu package install contour --package-name contour.community.tanzu.vmware.com --version 1.19.1
-```
+## What This Package Does
 
-After the Contour package has been installed, you can install Knative-Serving.
+Knative Serving provides components that enable rapid deployment of serverless containers and autoscaling of pods.
```

Comment:
Add hyphen in "Knative Serving" to be consistent with references elsewhere in the doc